src/dotnet/Telemetry.cs

```diff
@@ -19,6 +20,8 @@ public class Telemetry : ITelemetry
         private Dictionary<string, double> _commonMeasurements = null;
         private Task _trackEventTask = null;
 
+        internal static readonly string TelemetrySentinel  = Path.Combine(ApplicationEnvironment.ApplicationBasePath, $"{Product.Version}.dotnetTelemetry");
```

Comment:
If the first time experience is skipped (via` DOTNET_SKIP_FIRST_TIME_EXPERIENCE`), and the message doesn't get shown, should we still enable telemetry?